x-pack/plugins/ingest_manager/public/plugin.ts

```diff
@@ -55,13 +58,30 @@ export class IngestManagerPlugin
           IngestManagerStartDeps,
           IngestManagerStart
         ];
+        console.log('plugin#setup mount()');
         const { renderApp } = await import('./applications/ingest_manager');
         return renderApp(coreStart, params, deps, startDeps, config);
       },
     });
   }
 
-  public start(core: CoreStart) {}
+  public async start(core: CoreStart): Promise<IngestManagerStart> {
+    const config = this.config;
+    core.http.post('/api/ingest_manager/setup');
```

Comment:
probably missing an await here

Reply:
Great catch! I think this was the cause of the errors but will confirm